### PR TITLE
Add slugified name in report urls

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Relate #116

### CODE REVIEW

File: thinkhazard/models.py [hunk 1]
```diff
@@ -259,16 +260,22 @@ class AdministrativeDivision(Base):
...
         if self.leveltype_id == 3:
             return {'code': self.code,
                     'admin0': self.parent.parent.name,
                     'admin1': self.parent.name,
-                    'admin2': self.name}
+                    'admin2': self.name,
+                    'url': request.route_url('report_overview', division=self)}
+
+    def slug(self):
+        return slugify(self.name)
```
Comment (pgiraud) [inline thinkhazard/models.py]: I think the name of parent divisions should also be present.

**Comment by pgiraud** [comment]
The zoom button on map or the parent link in the breadcrumb point to a 404 page.

**Comment by arnaud-morvan** [comment]
Done

**Comment by pgiraud** [comment]
LGTM. Please merge.

**Pull request merged**